PrintNodePrintJobOptions.cs

```diff
@@ -64,7 +64,7 @@ public class PrintNodePrintJobOptions
         /// See <a href="https://www.printnode.com/docs/api/curl/#printer-capabilities">Printer capabilities</a>.
         /// </summary>
         [JsonProperty("nup")]
-        public int Nup { get; set; }
+        public int Nup { get; set; } = 1;

```

Comment:
According to the official docs this should be an array of integers, so we might have another issue here. What's the reasoning/result of defaulting this to 1?

Reply:
The print capabilities return an array for Nup but for the print, it asks
for an Integer. The default they document is 1.